Project.toml

```diff
@@ -0,0 +1,25 @@
+name = "InfrastructureModels"
+uuid = "2030c09a-7f63-5d83-885d-db604e0e9cc0"
+version = "0.1.0"
+
+[deps]
+JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
+Memento = "f28f55f0-a522-5efc-85c2-fe41dfb9b2d9"
+
+[compat]
+JSON = ">= 0.6"
+JuMP = "< 0.19"
+Juniper = "< 0.4"
+Memento = "< 0.11"
+julia = "^1"
```

Comment:
Do entries in the `[compat]` apply to entries in `[extras]`? Or is the version requirement for Juniper just inherited from the requirement for JuMP?

Reply:
I am not sure, but adding just in case.